### ISSUE TITLE
Add Review Order option of "most recently due"

### ISSUE THREAD

**Issue by curiousjp** [davidsansome/tsurukame#173]
Hi - my use case is having a large backlog of reviews after taking a long time off. WK recommends I don't start over, but just keep hacking on my backlog until I clear it. 

I find myself in a situation where I encounter a card I don't recognize (in some cases effectively having to re-learn it), and it is then scheduled for review in (e.g.) 6 hours. But because of the large number of reviews, I don't see it again for a long time. This makes it hard to consolidate, which I would prefer to prioritize before reviewing further 'new' cards. 

Would it be possible to have an option to sort review order by most recently falling due? I thought Review Order: "SRS level order" might do something like this, but I think that is instead working on the apprentice / guru / master / enlightened / burned hierarchy.

**Comment by turtlemaster19** [comment]
It wouldn't actually be scheduled for review in 6 hours, since WaniKani's SRS doesn't work that way. You are correct that "SRS level order" is:
> working on the apprentice / guru / master / enlightened / burned hierarchy.

This means that the reviews that had an interval of 4 hours, then 8 hours, etc. would come.
But, when you say:
> sort review order by most recently falling due

What is "falling due" and when should it show the "further 'new' cards"? I completely agree that it could be useful, but as a programmer myself, we would need to know how to implement this.

**Comment by curiousjp** [comment]
Sorry, perhaps I explained this poorly. My understanding is that the Assignment data structure in the V2 API contains an "available_at" date when a particular item will become available to be reviewed. 

Roughly, I was suggesting that it should be possible to sort the review queue so that items with the most recent available_at date were at the front - so that an item that has been in my review queue already for a very long time can stay there a little longer while I work on consolidating the items I have reviewed lately.  It is not so important to me now that I have worked through my backlog. :)
